fake_service/schema/utils.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+
+from faker import Faker
+from typing import Any, Dict, List
+from faker.generator import random
+
+fake = Faker()
+
+
+def make_generetions(data: List, count: str) -> List:
+    list2 = []
+    for i in range(int(count)):
+        dict1 = {}
+        for item in data:
+            name = item["name"]
+            if item["type"] == "integer":
+                dict1[name] = random.randint(
+                    int(item["value_from"]), int(item["value_to"])
+                )
+            if item["type"] != "integer":
+                column_type = getattr(fake, item["type"])
+                dict1[name] = column_type()
+        list2.append(dict1)
+    return list2
```

Comment:
Use better namings, your variables should tell whats inside by theirs name. `list2`, `dict1`   don't tell me that information. Check out this [video](https://www.youtube.com/watch?v=hjdRQspuelk) about namings . Also use proper typings. FE: `data` argument in this function has "List" type. List of what entities? List of dicts? if so You can use List[Dict[str, Any]], etc. Check out this [doc](https://peps.python.org/pep-0484/) about typings. Also I don't understand purpose of this function.